kyaml/fieldmeta/fieldmeta.go

```diff
@@ -254,6 +274,8 @@ const (
 // shortHandRef is the shorthand reference to setters and substitutions
 var shortHandRef = "$openapi"
 
+var SetterVersion = "v2"
```

Comment:
Can we avoid this? This seems like something that should be a function parameter that we have moved to a package level variable. This makes it much harder to use the library in other situations.

Reply:
I felt it's okay for it to be package level variable as the setter version is constant per command invocation. But avoided it and updated the PR. PTAL.